src/app/frontend/common/components/list/column/menu/template.html

```diff
@@ -38,15 +38,18 @@
           (click)="isPinned() ? onUnpin() : onPin()"
           i18n>{{ isPinned() ? "Unpin" : "Pin" }}</button>
   <button mat-menu-item
+          data-cy="edit"
```

Comment:
Adding framework specific attributes seems a bit weird to me. You should use `id` in such cases.

@maciaszczykm WDYT?

Reply:
I agree.

Reply:
Cypress has some best practices here regarding element selection: https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements.

That's what I tried to follow.

Reply:
Yes, but it is very cypress specific. If we'd like to move to another framework we'll have all this `data-cy` attributes that we do not need. We should always use something generic.

Reply:
Okay, so should we rename `data-cy` to something else? Like `kd-test`...

Reply:
No, just use IDs, like `<div id="...">...</div>`.

Reply:
Okay, will update.

Reply:
Done.